### PR TITLE
Updating ruby version

### PR DESCRIPTION
**Pull request by cherchezlafemme**
- Changed Ruby version from 2.2.1 to 2.3.1
- Updated Readme with instructions of using rvm and rbenv
- Added information to Readme on seeding db (was missing this step)
- Added $ to all code blocks in order to be consistent across the document (We had some with $, some without)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,41 +15,82 @@ Yes! *She's Coding* is build by the community, for the community. We currently h
...
+      $ rvm use 2.3.1
+      $ \curl -sSL https://get.rvm.io | bash -s stable
+      ```
+
+    ##### Set ruby version if you already have rvm installed:
+      + Select this version for use:
+
+      ```
+      $ rvm use 2.3.1
+      $ \curl -sSL https://get.rvm.io | bash -s stable
```
Comment (natsteinmetz) [inline README.md]: If we already have rvm installed, we won't need to get rvm via curl?

File: README.md [hunk 2]
```diff
@@ -15,41 +15,82 @@ Yes! *She's Coding* is build by the community, for the community. We currently h
...
-
-+ Select this version for use:
-  ```
-  rvm use 2.2.1
   ```
+  $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  ```
+
++ Set the [Ruby](https://www.ruby-lang.org/en/) version to 2.3.1. In order to be able to pick the ruby version or to download ruby, you will need to use Ruby version manager. There are 2 ruby version management tools: [rvm](https://rvm.io/) and [rbenv](https://github.com/rbenv/rbenv).
+
```
Comment (natsteinmetz) [inline README.md]: Could you change the 'you will need to use Ruby version manager' to 'we recommend using a ruby version manager'? Thank you!

Comment (cherchezlafemme) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (cherchezlafemme) [inline README.md]: sorry, missed that. deleted curl part. you don't need it

**Comment by cherchezlafemme** [comment]
Updated branch to reflect requested changes

**Pull request merged**